yarn-error.log

```diff
@@ -0,0 +1,7776 @@
+Arguments: 
```

Comment:
I guess this was an accidental commit.

Can you please delete this and add it to the `.gitignore` file please?

Reply:
good catch, thanks!